arm/proofs/arm.ml

```diff
@@ -922,14 +946,21 @@ let ARM_ADD_RETURN_STACK_TAC =
    `(!x. (!y. P x y) ==> (!y. Q x y)) ==> (!x y. P x y) ==> (!x y. Q x y)`
   and sp_tm = `SP` and x30_tm = `X30`
   and dqd_thm = WORD_BLAST `(word_zx:int128->int64)(word_zx(x:int64)) = x` in
-  fun execth coreth reglist stackoff ->
+  fun ?(pre_post_nsteps:(int*int) option) execth coreth reglist stackoff ->
```

Comment:
I had to add this optional argument to `ARM_ADD_RETURN_STACK_TAC` because the previous heuristics for determining the number of instructions used to do callee-save register save/restore did not match the cases in p384_montjadd/double.S . Since this is an optional argument, the existing proofs simply sets it to None and hence has no effect.